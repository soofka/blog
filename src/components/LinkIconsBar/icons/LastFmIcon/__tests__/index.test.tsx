import * as React from 'react';
import { shallow } from 'enzyme';

import LastFmIcon from 'components/LinkIconsBar/icons/LastFmIcon';

describe('<LastFmIcon/>', () => {

  it('renders correctly', () => {
    expect(shallow(<LastFmIcon/>)).toMatchSnapshot();
  });

});
