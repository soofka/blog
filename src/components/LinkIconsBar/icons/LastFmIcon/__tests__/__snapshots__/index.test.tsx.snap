exports[`<LastFmIcon/> renders correctly 1`] = `
<a
  href="https://www.last.fm/user/soofka"
  target="_blank">
  <svg
    viewBox="0 0 512 512">
    <path
      d="M230.104 336.568l-13.607-36.988s-22.11 24.66-55.268 24.66c-29.341 0-50.172-25.512-50.172-66.328 0-52.293 26.359-71.001 52.297-71.001 37.412 0 49.316 24.234 59.522 55.273l13.607 42.518c13.603 41.236 39.113 74.402 112.666 74.402 52.727 0 88.437-16.155 88.437-58.672 0-34.438-19.56-52.297-56.125-60.802l-27.209-5.951c-18.707-4.252-24.233-11.906-24.233-24.659 0-14.456 11.478-22.96 30.189-22.96 20.406 0 31.458 7.653 33.162 25.935l42.516-5.103c-3.402-38.263-29.761-53.996-73.127-53.996-38.266 0-75.68 14.456-75.68 60.799 0 28.912 14.029 47.197 49.315 55.697l28.916 6.801c21.683 5.104 28.908 14.031 28.908 26.363 0 15.73-15.305 22.107-44.218 22.107-42.941 0-60.794-22.534-70.999-53.574l-14.032-42.513c-17.854-55.271-46.342-75.68-102.892-75.68-62.499-.001-95.663 39.538-95.663 106.715 0 64.628 33.164 99.489 92.689 99.489 48.038 0 71.001-22.532 71.001-22.532z" />
  </svg>
</a>
`;

exports[`<LastFmIcon/> renders correctly with optional props 1`] = `
<a
  href="https://www.last.fm/user/soofka"
  target="_blank">
  <svg
    className="testClassName"
    viewBox="0 0 512 512">
    <path
      d="M230.104 336.568l-13.607-36.988s-22.11 24.66-55.268 24.66c-29.341 0-50.172-25.512-50.172-66.328 0-52.293 26.359-71.001 52.297-71.001 37.412 0 49.316 24.234 59.522 55.273l13.607 42.518c13.603 41.236 39.113 74.402 112.666 74.402 52.727 0 88.437-16.155 88.437-58.672 0-34.438-19.56-52.297-56.125-60.802l-27.209-5.951c-18.707-4.252-24.233-11.906-24.233-24.659 0-14.456 11.478-22.96 30.189-22.96 20.406 0 31.458 7.653 33.162 25.935l42.516-5.103c-3.402-38.263-29.761-53.996-73.127-53.996-38.266 0-75.68 14.456-75.68 60.799 0 28.912 14.029 47.197 49.315 55.697l28.916 6.801c21.683 5.104 28.908 14.031 28.908 26.363 0 15.73-15.305 22.107-44.218 22.107-42.941 0-60.794-22.534-70.999-53.574l-14.032-42.513c-17.854-55.271-46.342-75.68-102.892-75.68-62.499-.001-95.663 39.538-95.663 106.715 0 64.628 33.164 99.489 92.689 99.489 48.038 0 71.001-22.532 71.001-22.532z" />
  </svg>
</a>
`;
